dev/NavigationView/NavigationView.xaml

```diff
@@ -130,6 +130,17 @@
                                     </VisualState.Setters>
                                 </VisualState>
                             </VisualStateGroup>
+
+                            <VisualStateGroup x:Name="PaneVisibilityGroup">
+                                <VisualState x:Name="PaneVisible" />
+                                <VisualState x:Name="PaneCollapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="RootSplitView.DisplayMode" Value="CompactOverlay" />
```

Comment:
>DisplayMode [](start = 70, length = 11)

NavigationView.cpp changed DisplayMode from code heavily, please make sure it will not break any regression.

Reply:
I hadn't noticed any regressions from the test runs yet but if there's writing from code then yeah there could be a conflict. Looks like NavigationView::UpdateVisualStateForDisplayModeGroup is the one place that writes it to the SplitView. Think I should move this from markup into UpdateVisualStateForDisplayModeGroup or the reverse?

Reply:
Moved to code with the latest commit.